doc/modules/classes.rst

```diff
@@ -915,11 +915,20 @@ See the :ref:`metrics` section of the user guide for further details.
    metrics.pairwise.pairwise_kernels
    metrics.pairwise.polynomial_kernel
    metrics.pairwise.rbf_kernel
+   metrics.pairwise.sigmoid_kernel
+   metrics.pairwise.cosine_similarity
+   metrics.pairwise.cosine_distances
    metrics.pairwise.laplacian_kernel
    metrics.pairwise_distances
    metrics.pairwise_distances_argmin
    metrics.pairwise_distances_argmin_min
-
+   metrics.pairwise.check_pairwise_arrays
+   metrics.pairwise.check_paired_arrays
+   metrics.pairwise.paired_euclidean_distances
```

Comment:
@MechCoder noted. Will revert it. But how should i know if a method is meant for public use or not ?